### DISCUSSION TITLE
Local solar time

### DISCUSSION THREAD

**Discussion by pad39b** [Stellarium/stellarium#1167] [Q&A] [answered]
I can't find a way to see permanently the local solar time, that is, a clock showing 12:00:00 when the sun is at transit. Maybe it is not implemented?

I would like to see the solar time on any location selected. 

Maybe it could be added next to the standard time clock and/or with an option to turn it on/off.

I consider the solar time useful to notice the difference with the standard time not only at sun transit time.

Hope this can be implemented.
Thank you.

**Comment by alex-w** [comment, marked as answer]
You may enable LMST or LTST time zones on Location window to get a local mean solar time or local true solar time respectively.

**Comment by gzotti** [comment]
You can also show equation of time with the plugin of the same name.

**Comment by alex-w** [comment]
Equation of time show difference between LMST and LTST

**Comment by gzotti** [comment]
Yes, but this may also be of interest in this context.

**Comment by pad39b** [comment]
Oh! I see, didn't know there were time zones for LMST and LTST! thank you alex-w, and thank you gzotti, it is also useful for me to know that :)

**Comment by pad39b** [comment]
On the object information it is displayed the Sidereal time. Would be nice to see there too (below the apparent sidereal time and above the rise time) the LTST so it is always visible when selecting any object, as it is already with the sidereal time.

**Comment by gzotti** [comment]
I know the importance of sidereal time for observers. But what is the practical application for LTST today? What's next? Italian hours, Babylonian hours, Unequal hours?

**Comment by pad39b** [comment]
Ok, I just thought of solar time as a concept for education purposes, no worries. I will take it then from the time zone on the Location window. Thank you.
